src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -456,7 +456,7 @@ private function get_post_excerpt( $event_id, $import_excerpt ) {
 		if ( $event_id ) {
 			$post_excerpt = get_post( $event_id )->post_excerpt;
 
-			return empty( $post_excerpt ) && ! empty( $import_excerpt ) ? $import_excerpt : $post_excerpt;
+			return !empty( $import_excerpt ) ? $import_excerpt : $post_excerpt;
```

Comment:
Small suggestion to remove the need to use a negation operator here.

```suggestion
			return empty( $import_excerpt ) ? $post_excerpt : $import_excerpt;
```

Reply:
Another question about this one, here we are assuming `$post_excerpt` is not empty, is there any other check before this one to ensure the returned value is not empty in case `$post_excerpt` is not empty